README.md

```diff
@@ -3521,7 +3521,7 @@ Other Style Guides
     ];
 
     // also good
-    const httpRequests = [
+    const httpUrlRequests = [
```

Comment:
first, let's keep `httpRequests`; but second, `Url` is literally never correct, ever, and should never appear in your code, so this would be a very bad example to provide.

Reply:
`Url` is correct in this case in Google's TypeScript, JavaScript, and C++ style guides

https://google.github.io/styleguide/tsguide.html#camel-case
https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Reply:
Unfortunately Google's style guides haven't ever been considered idiomatic by the wider ecosystem.